node.js/index.js

```diff
@@ -0,0 +1,114 @@
+// External Libraries:
+const agent = require('agentkeepalive');
+const request = require('request');
+
+// Internal Libraries:
+const constants = require('./constants');
+
+// Getting Environment Variables:
+const getConfig = () => {
+    var config = {};
+
+    if (process.env.LOGDNA_KEY) config.key = process.env.LOGDNA_KEY;
+    if (process.env.LOGDNA_HOSTNAME) config.hostname = process.env.LOGDNA_HOSTNAME;
+
+    if (process.env.LOGDNA_TAGS && process.env.LOGDNA_TAGS.length > 0) {
+        config.tags = process.env.LOGDNA_TAGS.split(',').map(tag => tag.trim()).join(',');
+    }
+
+    return config;
+};
+
+// Message Sanity Check:
+const sanitizeMessage = message => {
+    var sanitized = message.substring(0, constants.MAX_LINE_LENGTH);
+    if (sanitized !== message) sanitized += ' (cut off, too long...)';
+    return sanitized;
+};
+
+// Parsing the gzipped Log Data:
+const parseEvent = event => {
+    return JSON.parse(require('zlib').unzipSync(Buffer.from(event.awslogs.data, 'base64')));
+};
+
+// Preparing the Messages and Options:
+const prepareLogs = eventData => {
+    return eventData.logEvents.map(event => {
+        return {
+            line: JSON.stringify({
+                message: sanitizeMessage(event.message)
+                , source: 'CloudWatchLogs'
+                , event: {
+                    type: eventData.messageType
+                    , id: event.id
+                }, log: {
+                    group: eventData.logGroup
+                    , stream: eventData.logStream
+                }
+            }), timestamp: event.timestamp
```

Comment:
Same as above, `,` should start on a new line?

Reply:
Done!